wger/manager/helpers.py

```diff
@@ -212,7 +212,7 @@ def get_reps_reprentation(setting, rep_unit):
         "Until Failure" unit
         """
         if setting.repetition_unit_id != 2:
-            reps = "{0} {1}".format(setting.reps, rep_unit).strip()
+            reps = "{0} {1} ({2})".format(setting.reps, rep_unit, setting.rpe).strip()
```

Comment:
We need to add a bit more of logic here when there's no value selected. So instead of "3 × 4 (None) (10 kg)", "3 × 4 (10 kg)" or 3 × 4 (10 kg 2RiR)"